src/bar/BarChartWithErrors.ts

```diff
@@ -58,6 +62,32 @@ export default class BarChartWithErrors extends mixins(Bar) {
                         if (tooltipItem.yLabel) {
                             label += formatCallback(tooltipItem.yLabel);
                         }
+
+                        interface ErrorBars {
+                            [xLabel: string]: {
+                                minus: number
+                                plus: number
+                            }
+                        }
+
+                        interface ChartDataSetsWithErrors extends ChartDataSets {
+                            errorBars?: ErrorBars
+                        }
+
+                        let minus = null
+                        let plus = null
+
+                        if (this.showErrors && tooltipItem.xLabel && typeof tooltipItem.datasetIndex !== "undefined" && data.datasets && data.datasets[tooltipItem.datasetIndex]) {
+                            const errorBars = (data.datasets[tooltipItem.datasetIndex] as ChartDataSetsWithErrors).errorBars
+                            if (errorBars && errorBars[tooltipItem.xLabel]) {
+                                minus = errorBars[tooltipItem.xLabel].minus
+                                plus = errorBars[tooltipItem.xLabel].plus
+                            }
+                        }
+                        if (minus && plus) {
+                            label = `${label} (${formatCallback(minus)} - ${formatCallback(plus)})`
```

Comment:
Ideally the range would use an en rule (–) rather than a hyphen but this old version of js doesn't seem to be able to compile that symbol.

Reply:
I think it's more likely an HTML issue than js - using &ndash doesn't seem to work though (tried that), possibly because it's being rendered onto a canvas via chart hs. I think a hyphen is fine!